src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,57 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.ByteArrayOutputStream
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  "printHelp" should "produce help info with values from custom properties file" in {
+
+    val result = executeWithHomeDir("src/test/resources")
+
+    // assert values that are neither hard coded nor in assembly/dist but in test/resources
+    result should include("127.0.0.1")
+    result should include("(default = secret)")
+    result should include("(default = somebody)")
+    result should include("src/test/resources")
+    result should include("${project.version}") // FIXME ?
+  }
+
+  it should "be pasted into README.md" in {
```

Comment:
While creative, I see little value in providing this "unit test". Copy-pasting the output of the `--help` option is a neat trick but can just as well be done by running the application and copy-pasting from the terminal screen.